### PR TITLE
REF-14 fixed non static menubar on keyboard pop up; fixed text hyperlink

### PR DESCRIPTION
**Pull request by jmarecaux**
menubar doesn't shift now when the keyboard is being used

the checkout without flight text hyperlink was only working if the tiny chevron sign was pressed so i changed it to make the underline text link there instead and updated some formatting

### CODE REVIEW

File: src/components/main/HomeScreen.js [hunk 1]
```diff
@@ -215,8 +212,10 @@ export default class HomeScreen extends React.Component {
                 </View>
               </View>
             </View>
-            <MenuBar navigation={this.props.navigation} />
           </ImageBackground>
+          <MenuBar navigation={this.props.navigation} />      
+            <MenuBar navigation={this.props.navigation} />
+          <MenuBar navigation={this.props.navigation} />
```
Comment (wang1kevin1) [inline src/components/main/HomeScreen.js:218]: bruh what. You added 3?

File: src/components/main/HomeScreen.js [hunk 2]
```diff
@@ -160,27 +160,24 @@ export default class HomeScreen extends React.Component {
...
+                      doe sem número de vôo
+                    </Text>
+                    {" "}
                     <MaterialCommunityIcons
                       name="chevron-double-right"
                       style={styles.chevronIcon}
-                      onPress={() =>
-                        this.props.navigation.navigate("CheckoutWithoutFlight")
-                      }></MaterialCommunityIcons>
+                    />
```
Comment (wang1kevin1) [inline src/components/main/HomeScreen.js:178]: both the text and the chevrons should be able to navigate

File: src/components/main/HomeScreen.js (hunk 1 above)
Comment (jmarecaux) [reply src/components/main/HomeScreen.js:218]: lol. idk why that happened i must've accidentally undo'd something before pushing

**Comment by jmarecaux** [comment]
> HomeScreen isn't the only one where the menubar messes up.
> 
> In general, all screens have a shifting menubar atm.

which other screens? I looked through the app on master, i could only find two current instances where the keyboard is being used, the home screen fixed and the sign up screen was already not shifting

**Comment by wang1kevin1** [comment]
Okay, what I mean by all screens is all screens that have text input. So yea, just Homescreen and SignUp screen.

It also happens in REF-8 Sign in screen on android if you note the picture I attached to the pull request

**Comment by jmarecaux** [comment]
got it, should be good now and i can fix the menubar shift inside ref8 pr i think

**Comment by wang1kevin1** [comment]
Remember to fix linking in signin screen too. both text and chevrons need linking

**Comment by wang1kevin1** [comment]
Okay this doesn't work on android. However I think I found a fix:

Please change
**MenuBar.js**
`container: {
        position: "absolute",
        left: 0,
        bottom: 0, ...`

to

`container: {
        position: "absolute",
        left: 0,
        top: 0, ...`

and see how it works

**Pull request merged**